netnode/netnode.py

```diff
@@ -175,16 +185,16 @@ def _strset(self, key, value):
         if len(value) > BLOB_SIZE:
             storekey = self._get_next_slot(STR_KEYS_TAG)
             self._n.setblob(value, storekey, STR_KEYS_TAG)
-            self._n.hashset(key, str(storekey), STR_TO_INT_MAP_TAG)
+            self._n.hashset(key, struct.pack('>I', storekey), STR_TO_INT_MAP_TAG)
```

Comment:
we've just found that this introduced a breaking change in how we're storing data. this means that .idb files manipulated by ida-netnode prior to the PR are incompatible with the new library version.

will probably have to introduce more code to attempt to detect this scenario and maintain compatibility. then, will release a new major version of the library without the compat mode.

Reply:
alternatively, revert this change and keep the storekey as ASCII-encoded decimal number. then v2.0.x will be considered broken. we can release v3.0.0 with reverted behavior.

Reply:
ultimately, i prefer the packed bytes method, but we have a bunch of existing code and IDBs that this library needs to continue to support. so starting to lean towards a revert.